.github/workflows/test-pip.yml

```diff
@@ -118,9 +136,7 @@ jobs:
         if: ${{ matrix.os == 'windows-2019' }}
 
       - name: Build PyGEOS
-        run: |
-          python setup.py build_ext --inplace
-          pip install --no-deps -e .
+        run: python setup.py build_ext --inplace
```

Comment:
Is there a reason to no longer install the package? (because we are running the tests from the source directory anyway?)

Reply:
Yes there is no reason to “pip install” if you are in the source directory and if the extensions are built. Removing the line saves some seconds in the CI.